hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java

```diff
@@ -253,9 +253,9 @@ public Optional<Throwable> execFor(final HapiSpec spec) {
                 return Optional.empty();
             }
             if (verboseLoggingOn) {
```

Comment:
Why do we need a verboseLoggingOn flag? The log.properties should specify whether logging is enabled and at what logging level.

Reply:
This comment applies to a large number of other classes in this PR